Image Scrapper

```diff
@@ -0,0 +1,34 @@
+from bs4 import BeautifulSoup
+import requests
+import urllib.request
+
+from google.colab import drive
+drive.mount('/content/drive')
+
+import pandas as pd
+df = pd.read_csv('/content/drive/MyDrive/Ralph_Lauren_Retail_Feed.csv')
```

Comment:
Can you make a final variable for base path at the top instead of static colab import? The final variable can be static too, but that matches the pattern we've been using elsewhere.